scripts/build_llvm_project.py

```diff
@@ -26,27 +26,29 @@ def get_arguments():
   buildopt.add_argument('-t', '--target', metavar='ARCH', choices=['X86', 'AArch64', 'PowerPC'], default='X86',
                         help='Control which targets are enabled (%(choices)s) (default: %(default)s)')
   buildopt.add_argument('-p', '--install-prefix', metavar='PATH', nargs='?', default=None, const=False,
-                        help='Install directory (default: do not install)')
+                        help='Set install directory (default: platform specific directory)')
+  buildopt.add_argument('-i', '--install', action='store_true', default=False,
+                        help='Do llvm install (default: do not install llvm)')
```

Comment:
Please reword these two help messages:
- `Specify installation directory (default: platform-specific location)`
- `Install LLVM  (default: do not install)`

Reply:
Thanks for the review and the suggestion. Updated.